_posts/2018-10-29-bazel-0.19.0.md

```diff
@@ -0,0 +1,91 @@
+
+* Runfiles in `cc_test`: the C++ runfiles library (`@bazel_tools//tools/cpp/runfiles`) can now create Runfiles objects for tests. See `//tools/cpp/runfiles/runfiles_src.h` in the Bazel source tree for documentation.
+
+* :cc_binary link action no longer hardcodes `-static-libgcc` for toolchains that support embedded runtimes (guarded by `--experimental_dont_emit_static_libgcc` temporarily).
+
+* The flag `--experimental_enable_cc_configuration_make_variables` is removed, use `--incompatible_disable_cc_configuration_make_variables` instead.
+
+## Java
+
+* Bazel now requires a JDK to be installed as it no longer falls back to the internal JDK if the `--javabase` flag is not set. Instead, Bazel now searches for a JDK within the `JAVA_HOME` and `PATH` environment variables. If Bazel cannot find a JDK, it will not be able to run Java builds. 
+
+## Code Coverage
+
+* LcovMerger was renamed to [CoverageOutputGenerator](https://github.com/bazelbuild/bazel/pull/6137). Please use 
+```
+--coverage_report_generator=@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main
+```
+
+instead of the previous 
+
+```
+--coverage_report_generator=@bazel_tools//tools/test/LcovMerger/java/com/google/devtools/lcovmerger:Main
+```
+
+
+* You can now enable a new coverage method for C++ rules that use gcc with the --experimental_cc_coverage flag. The flag will use gcov instead of lcov to collect coverage information. This new implementation fixes
+some of the C++ coverage correctness issues ( [respecting `--instrumentation_filter` and `--instrument_test_targets` flags](https://github.com/bazelbuild/bazel/issues/6129) ). 
+There have been observed faster clean and incremental times when invoking bazel coverage ( [small overhead compared to test runtimes instead of 2x test runtime](https://github.com/bazelbuild/bazel/issues/5882) ),  but more measurements need to be done.
+
+
+## Other Changes
+
+* Add `--apple_compiler` and `--apple_grte_top options`. These provide the equivalent of `--compiler` / `--grte_top` for the toolchain configured in `--apple_crosstool_top`.
+
+* There is now a `same_pkg_direct_rdeps` query function. See the query documentation for more details.
+
+* Propagating remote errors to the user even if `--verbose_failures=false` is set.
+
+* Add number of configured targets to analysis phase status output.
+
+* Bazel will now check stderr instead of stdout to decide if it is outputting to a terminal.  `--isatty` is deprecated, use `--is_stderr_atty` instead.
+
+## Future Changes
+
+We will do incompatible changes in Bazel 0.20. We encourage you to migrate in advance (or report any migration difficulty) with the following flags:
+
+    * [`--incompatible_disable_late_bound_option_defaults`](https://docs.bazel.build/versions/master/skylark/backward-compatibility.html#disable-late-bound-option-defaults)
```

Comment:
Based on the release page rendering (https://github.com/bazelbuild/bazel/releases/tag/0.19.0), I suggest to remove indentation before the `*`.

Reply:
Fixed.